package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@toyota-research-institute/lakefront",
   "description": "React UI Components",
-  "version": "0.54.3",
+  "version": "0.56.3",
```

Comment:
nit: shouldn't we reset the patch version to 0? (as in `0.56.0`?

Reply:
It gets patched automatically by the git workflow each time I make an update (like to address PR comments). If it seems like a huge issue, you can create an issue for us to rework the github workflow, but it might be slightly difficult to implement.

Maybe the better solution would be to create a seperate flow that locks the number to a 0 once a PR has been approved and/or we create a new npm release. It still might be difficult, but probably not as hard as getting these automatic patches to work a little better.